tests/testfiles/set_purpose_response_default.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ok": true,
+    "purpose": PURPOSE_PH
```

Comment:
This should be in `""`?

Reply:
No, it's a placeholder which will get replaced with a json dump later on, so no quotes allowed here.